ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/DefaultErrorMessages.java

```diff
@@ -19,12 +19,27 @@
 
 public class DefaultErrorMessages implements ErrorMessages {
 
-  public static final String COMMAND_RUNNER_DEGRADED_ERROR_MESSAGE =
+  public static final String COMMAND_RUNNER_DEGRADED_INCOMPATIBLE_COMMANDS_ERROR_MESSAGE =
       "The server has encountered an incompatible entry in its log "
           + "and cannot process further DDL statements."
           + System.lineSeparator()
           + "This is most likely due to the service being rolled back to an earlier version.";
 
+  public static final String COMMAND_RUNNER_DEGRADED_BACKUP_CORRUPTED_ERROR_MESSAGE =
+      "The server has detected that the command topic may be corrupted. The backup of the "
+          + "command topic does not match the current contents of command topic."
+          + System.lineSeparator()
+          + "DDL statements will not be processed until either:"
+          + System.lineSeparator()
+          + "1. The current command topic is deleted and the backup file is used "
+          + "to restore the command topic."
+          + System.lineSeparator()
+          + "2. The current backup file is deleted."
```

Comment:
All backups must be deleted, otherwise ksql won't start because another backup will be out of sync with the command topic.